definition/rpc.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2020 Caicloud Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package definition
+
+// RPCDescriptor describes a descriptor for API definition in RPC style.
+type RPCDescriptor struct {
+	Prefix string
+	// Description describes the usage of the path.
+	Description string
+	// Middlewares contains path middlewares.
+	Middlewares []Middleware
+	// Tags indicates tags of current definitions and child definitions.
+	// It will override parent descriptor's tags.
+	Tags []string
+	// Consumes indicates content types that current definitions
+	// and child definitions can consume.
+	// It will override parent descriptor's consumes.
+	Consumes []string
+	// Produces indicates content types that current definitions
+	// and child definitions can produce.
+	// It will override parent descriptor's produces.
+	Produces []string
```

Comment:
// Actions contain actions in this descriptor. These actions will inherit the Middlewares, Tags, Consumes, Produces of the descriptor if values in the action are not specified